### PR TITLE
Show list description

### PR DESCRIPTION
**Pull request by jwalgran**
## Overview

We currently allow contributors to specify a detailed description for a list but prior to this PR we were not displaying that description publicly. From the facility detail page we link to the profile page of any contributors associated with that facility so the profile page is a suitable place to display the facility lists and their descriptions.

Connects #356

## Demo

Screen Shot 2019-05-22 at 2 57 10 PM

## Notes

Contributors have a dedicated UI for viewing their own lists so we are only showing the list details on public profile pages, not the profile page of the logged in contributor.

We have added descriptions to some of the fixture facility lists to allow for easier testing of facility list UI elements in both the presence and absence of a description value.

While reviewing the profile serialization I took the opportunity to eliminate some redundant database calls.

## Testing Instructions

* Reset the DB
  * `./scripts/manage resetdb`
  * `./scripts/manage loadfixtures`
  * `./scripts/manage processfixtures`
* Make sure you are logged out
* Browse http://localhost:6543/profile/2 and verify that the facility list description is displayed.
* Browse http://localhost:6543/profile/3 and verify that the page displays normally without a facility list description
* Register and confirm an account
* Ensure you are not logged in, browse http://localhost:6543/profile/101, and verify that the Facility Lists section is not displayed. 

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

File: src/app/src/components/FacilityListSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        color: '#000 !important',
+        marginBottom: '24px',
+    }),
+});
+
+
+function FacilityListSummary({ name, description }) {
+    return (
+        <div>
+            <div style={facilityListSummaryStyles.nameStyles}>{name}</div>
```
Comment (kellyi) [inline src/app/src/components/FacilityListSummary.jsx]: Consider using `p` elements for these or the Material UI Typography component: https://material-ui.com/style/typography/

File: src/app/src/reducers/ProfileReducer.js [hunk 2]
```diff
@@ -135,6 +140,7 @@ export default createReducer({
             currentPassword: { $set: '' },
             newPassword: { $set: '' },
             confirmNewPassword: { $set: '' },
+            facilityLists: { $set: payload.facility_lists || [] },
```
Comment (kellyi) [inline src/app/src/reducers/ProfileReducer.js]: Consider using `|| initialState[profileSummaryFieldsEnum.facilityLists]`

File: src/app/src/components/FacilityListSummary.jsx (hunk 1 above)
Comment (jwalgran) [reply src/app/src/components/FacilityListSummary.jsx]: Good idea. Used some `<p>` tags in 00f77c4

File: src/app/src/reducers/ProfileReducer.js (hunk 2 above)
Comment (jwalgran) [reply src/app/src/reducers/ProfileReducer.js]: Good idea. Updated in f2d0eef

**Comment by jwalgran** [comment]
Thanks for the review.

**Pull request merged**